src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressionCommentFilter.java

```diff
@@ -149,36 +164,94 @@
+ *     //resume constant check
+ *
+ *     public static final int var3;
+ *     // violation above , Name 'var3' must match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'
+ *
+ *     //stop constant check
+ *     public static final int var4; // suppressed violation
+ *     //resume constant check
+ *
+ *     public void method1() {
+ *
+ *         try{
+ *
+ *         }catch(Exception ex){ // violation , Catching 'Exception' is not allowed
+ *
+ *         }
+ *
+ *         //stop constant check
+ *         try{
+ *
+ *         }catch(Exception ex){ // violation , Catching 'Exception' is not allowed
+ *
+ *         }
+ *         //resume constant check
+ *     }
+ * }
  * </pre>
  * <p>
  * To configure a filter so that {@code UNUSED OFF: <i>var</i>} and
  * {@code UNUSED ON: <i>var</i>} marks a variable or parameter known not to be
- * used by the code by matching the variable name in the message:
+ * used by the code by matching the variable name in the message through a
+ * specified message in messageFormat:
  * </p>
  * <pre>
  * &lt;module name="SuppressionCommentFilter"&gt;
- *   &lt;property name="offCommentFormat" value="UNUSED OFF\: (\w+)"/&gt;
- *   &lt;property name="onCommentFormat" value="UNUSED ON\: (\w+)"/&gt;
- *   &lt;property name="checkFormat" value="Unused"/&gt;
- *   &lt;property name="messageFormat" value="^Unused \w+ '$1'.$"/&gt;
+ *     &lt;property name="offCommentFormat" value="ILLEGAL OFF\: (\w+)"/&gt;
+ *     &lt;property name="onCommentFormat" value="ILLEGAL ON\: (\w+)"/&gt;
+ *     &lt;property name="checkFormat" value="IllegalCatch"/&gt;
+ *     &lt;property name="messageFormat" value="^Catching '$1' is not allowed.$"/&gt;
  * &lt;/module&gt;
+ * &lt;module name="MemberName"/&gt;
+ * &lt;module name="ConstantName"/&gt;
+ * &lt;module name="IllegalCatch"/&gt;
  * </pre>
  * <pre>
- * private static void foo(int a, int b) // UNUSED OFF: b
+ * class InputSuppressionCommentFilter
  * {
- * System.out.println(a);
- * }
+ *     int VAR1; // violation , Name 'VAR1' must match pattern '^[a-z][a-zA-Z0-9]*$'
  *
- * private static void foo1(int a, int b) // UNUSED ON: b
- * {
- * System.out.println(a);
+ *     //ILLEGAL OFF: Exception
+ *     int VAR2; // violation , Name 'VAR2' must match pattern '^[a-z][a-zA-Z0-9]*$'
+ *     //ILLEGAL ON: Exception
+ *
+ *     public static final int var3;
+ *     // violation above , Name 'var3' must match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'
+ *
+ *     //ILLEGAL OFF: Exception
+ *     public static final int var4;
+ *     // violation above ,  Name 'var4' must match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'
+ *     //ILLEGAL ON: Exception
+ *
+ *     public void method1() {
+ *
+ *         try{
+ *
+ *         }catch(Exception ex){ // violation , Catching 'Exception' is not allowed
+ *
+ *         }
+ *
+ *         //ILLEGAL OFF: Exception
+ *         try{
+ *
+ *         }catch(Exception ex){ // suppressed violation
```

Comment:
>ILLEGAL OFF: Exception


please some different var value `ILLEGAL OFF: Exception` to show that it tries to suppress by Check Name + message, so mismatch by catch type matters.
```
try {}
catch(Exception exc) {} // suppressed
catch(Error exc) {} // violation
```

Reply:
Done.